### PR TITLE
Development

### PR DESCRIPTION
**Pull request by AliceHolznecht**
💄 style: Add global styles, import Roboto font, and layout styling

- 🖋️ Imported 'Roboto' font from Google Fonts
- 🌐 Applied global styles to reset margin, padding, and box-sizing
- 🔗 Styled anchor tags with no text decoration and white color
- 🎨 Styled icon elements with white color and pointer cursor
- 🖼️ Styled images with object-fit, object-position, and border-radius
- 🌐 Set background gradient and full height for html and body
- 📐 Defined grid layout for container with specific column sizes
- 🎨 Applied background gradient to container
- 🗂️ Styled sidebar with full height, background color, padding, flex layout, and transition effects
- 🖼️ Styled logo within sidebar with flex alignment and gap

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+i{
+    color: #fff;
+    cursor: pointer;
+}
+
+img{
+    object-fit: cover;
+    object-position: top;
+    border-radius: 8px;
+
```
Comment (samuelchristofel) [inline styles.css:25]: Missing closing curly bracket.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    flex-direction: column;
+    justify-content: space-between;
+    z-index: 10000;
+    transition: all 0.6s ease;
+}
+
+.container .sidebar .logo{
+    display: flex;
+    align-items: center;
+    gap: 6px;
```
Comment (samuelchristofel) [inline styles.css]: Missing closing curly bracket.

**Review by AliceHolznecht — COMMENTED** [review]
Gonna Fix it ASAP

**Review by samuelchristofel — APPROVED** [review]
Done. Thank you

**Pull request merged**